core/src/test/scala/stryker4s/Stryker4sTest.scala

```diff
@@ -17,15 +18,24 @@ import scala.util.Success
 
 class Stryker4sTest extends Stryker4sSuite with LogMatchers {
 
+  class TestMutantRunner(sourceCollector: SourceCollector)(implicit config: Config)
```

Comment:
Why not create a mock MutantRunner? That way you can (possibly) also test if it is being called with the correct values